console/src/main/scripts/plugins/accounts/ts/invitation.ts

```diff
@@ -0,0 +1,55 @@
+///
+/// Copyright 2015 Red Hat, Inc. and/or its affiliates
+/// and other contributors as indicated by the @author tags.
+///
+/// Licensed under the Apache License, Version 2.0 (the "License");
+/// you may not use this file except in compliance with the License.
+/// You may obtain a copy of the License at
+///
+///    http://www.apache.org/licenses/LICENSE-2.0
+///
+/// Unless required by applicable law or agreed to in writing, software
+/// distributed under the License is distributed on an "AS IS" BASIS,
+/// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+/// See the License for the specific language governing permissions and
+/// limitations under the License.
+///
+
+/// <reference path='accountsPlugin.ts'/>
+
+module HawkularAccounts {
+
+  export class InvitationController {
+    public static $inject = ['$log', '$routeParams', 'HawkularAccount', 'NotificationsService'];
+    public invitation:IInvitation;
+    public loading:boolean;
+    public success:boolean;
+
+    constructor(private $log:ng.ILogService,
+                private $routeParams:any,
+                private HawkularAccount:any,
+                private NotificationsService:INotificationsService) {
+
+      this.loading = true;
+      this.success = false;
+      HawkularAccount.OrganizationInvitation.update({token: $routeParams.token},
+        (response:IInvitation) => {
```

Comment:
Love to see these typed responses! yay! 
Otherwise the maintainers have to go find the rest document elsewhere to see what it should even return.